eng/pipelines/linux.yml

```diff
@@ -28,6 +28,7 @@ jobs:
             _dockerContainer: rhel7_container
             _buildScriptPrefix: ''
             _buildExtraArguments: ''
+            _publishTests: true
```

Comment:
This is used in both, official build and PR builds, so that is another reason why we need to if condition that step for official builds.

Reply:
Yes, I've added both conditions (submitToHelix/isOfficialBuild